.github/workflows/label_approved_veps.yaml

```diff
@@ -27,4 +27,5 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GH_PAT }}
           PR_NUMBER: ${{ github.event.pull_request.number }}
+          TARGET_PROJECT_URL: "https://github.com/orgs/kubevirt/projects/15"
```

Comment:
Should this be manually updated, or will there be an automation?

Reply:
Store and then pull the latest development release project ID from the sig-release project? Not required now obviously but would avoid updating this every release.